src/store/ducks/jurisdictions.ts

```diff
@@ -64,18 +97,36 @@ interface RemoveAllJurisdictionIdsAction extends AnyAction {
   allJurisdictionIds: {};
 }
 
+/** Remove childrenByParentId action interface */
+interface RemoveChildrenByParentIdAction extends AnyAction {
+  childrenByParentId: {};
+  type: typeof REMOVE_CHILDREN_BY_PARENT_ID;
+}
+
+/** Remove jurisdictionIdsByPlanId action interface */
+interface RemoveJurisdictionIdsByPlanId extends AnyAction {
+  jurisdictionIdsByPlanId: {};
+  type: typeof REMOVE_JURISIDCTIONS_BY_PLAN_ID;
+}
+
 /** jurisdiction action types */
 export type JurisdictionActionTypes =
   | FetchJurisdictionAction
   | FetchAllJurisdictionIdsAction
+  | FetchChildrenByParentIdAction
+  | FetchJurisdictionIdsByPlanId
   | RemoveJurisdictionsAction
   | RemoveAllJurisdictionIdsAction
+  | RemoveChildrenByParentIdAction
+  | RemoveJurisdictionIdsByPlanId
   | AnyAction;
 
 /** interface to describe jurisdiction state */
 interface JurisdictionState {
   allJurisdictionIds: AllJurisdictionIds;
-  jurisdictionsById: { [key: string]: Jurisdiction };
+  childrenByParentId: ChildrenByParentId;
+  jurisdictionIdsByPlanId: JurisdictionIdsByPlanId;
+  jurisdictionsById: { [key: string]: JurisdictionTypes };
```

Comment:
Why did we change from `Jurisdiction` to `JurisdictionTypes`?  I believe `JurisdictionTypes` is just a list of strings like ['administrative', 'operational']?

Reply:
Ah, poor naming convention on my part, I can rename but essentially `JurisdictionTypes` is being brought in from settings config file to allow `Jurisdiction`s with additional properties to be added to the same reducer.